sentry-spring-boot-jakarta/src/main/java/io/sentry/spring/boot/jakarta/SentryWebfluxAutoConfiguration.java

```diff
@@ -26,6 +27,7 @@
 import reactor.core.scheduler.Schedulers;
 
 /** Configures Sentry integration for Spring Webflux and Project Reactor. */
+@Autoconfiguration
```

Comment:
```suggestion
@AutoConfiguration
```

Reply:
I'm assuming we should change all our auto config classes to use this annotation in Spring Boot 3. There's a list of those in `org.springframework.boot.autoconfigure.AutoConfiguration.imports` (https://github.com/getsentry/sentry-java/blob/d00722594da449318b4cbcc7ce104d2bbc1c050b/sentry-spring-boot-jakarta/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports)

Reply:
@adinauer 
new commit applied to those files you mentioned too :)